alfresco-transform-tika/alfresco-transform-tika-boot/src/test/resources/quick.ods_metadata.json

```diff
@@ -1,6 +1,6 @@
-{
-  "{http://www.alfresco.org/model/content/1.0}description" : "Gym class featuring a brown fox and lazy dog",
-  "{http://www.alfresco.org/model/content/1.0}created" : 845336008000,
-  "{http://www.alfresco.org/model/content/1.0}author" : "Derek Hulley",
-  "{http://www.alfresco.org/model/content/1.0}title" : "The quick brown fox jumps over the lazy dog"
+{
+  "{http://www.alfresco.org/model/content/1.0}description" : "Gym class featuring a brown fox and lazy dog",
+  "{http://www.alfresco.org/model/content/1.0}created" : 845343208000,

```

Comment:
Please revert this file. ods is not handled by IPTC extraction so should not change.